react/features/recording/components/Recording/native/StartRecordingDialogContent.tsx

```diff
@@ -0,0 +1,339 @@
+/* eslint-disable lines-around-comment  */
+import React from 'react';
+import { Image, View } from 'react-native';
+import { Switch, Text } from 'react-native-paper';
+
+// @ts-ignore
+import { translate } from '../../../../base/i18n';
+// @ts-ignore
+import { LoadingIndicator } from '../../../../base/react';
+// @ts-ignore
+import { connect } from '../../../../base/redux';
+import Button from '../../../../base/ui/components/native/Button';
+import { BUTTON_TYPES } from '../../../../base/ui/constants';
+// @ts-ignore
+import { RECORDING_TYPES } from '../../../constants';
+// @ts-ignore
+import { getRecordingDurationEstimation } from '../../../functions';
+import AbstractStartRecordingDialogContent, {
+    type Props,
+    mapStateToProps
+} from '../AbstractStartRecordingDialogContent';
+import {
+    DROPBOX_LOGO,
+    ICON_CLOUD,
+    ICON_INFO,
+    ICON_USERS,
+    TRACK_COLOR
+    // @ts-ignore
+} from '../styles.native';
+
+
+/**
+ * The start recording dialog content for the mobile application.
+ */
+class StartRecordingDialogContent extends
+// @ts-ignore
+    AbstractStartRecordingDialogContent<Props> {
+    /**
+     * Renders the component.
+     *
+     * @protected
+     * @returns {React$Component}
+     */
+    render() {
+        // @ts-ignore
+        const { _styles: styles } = this.props;
+
+        return (
+            <View style = { styles.container }>
+                { this._renderNoIntegrationsContent() }
+                { this._renderFileSharingContent() }
+                { this._renderUploadToTheCloudInfo() }
+                { this._renderIntegrationsContent() }
+            </View>
+        );
+    }
+
+    _shouldRenderNoIntegrationsContent: () => boolean;
```

Comment:
Why are these needed? They should be removed in TS files

Reply:
TS2551: Property '_shouldRenderNoIntegrationsContent' does not exist on type 'StartRecordingDialogContent'. 
Same for the others

Reply:
You should be able to remove them now that the Abstract component issue is solved